core-war/src/main/java/org/silverpeas/web/jobdomain/control/JobDomainPeasSessionController.java

```diff
@@ -2137,4 +2135,27 @@ private UserDetail getUser(String userId, UserDetail[] users) {
     }
     return null;
   }
-}
+
+  private void processDataOfExtraTemplate(String userId, HttpRequest request) {
+    PublicationTemplateManager templateManager = PublicationTemplateManager.getInstance();
+    PublicationTemplate template = templateManager.getDirectoryTemplate();
+    if (template != null) {
+      try {
+        PagesContext context = getTemplateContext(userId);
+        templateManager.saveData(template.getFileName(), context, request.getFileItems());
+      } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1166)